documentation/modules/configuring/proc-migrating-clusters-node-pools.adoc

```diff
@@ -93,3 +93,12 @@ The resources remain identical to how they were before.
 
 . Remove the replicated properties from the `Kafka` custom resource.
 When the `KafkaNodePool` resource is in use, you can remove the properties that you copied to the `KafkaNodePool` resource, such as the `.spec.kafka.replicas` and `.spec.kafka.storage` properties.
+
+.Reversing the migration
+
+To revert to managing Kafka nodes using only `Kafka` custom resources:
+
+. Move all brokers (and partition replicas) to a single `KafkaNodePool` named `kafka` with node IDs from 0 to N (where N is the number of replicas).
```

Comment:
At the beginning this was misleading to me.
I was thinking "having node pools is the starting point and I want to reverse why I should ... Move all brokers to a single KafkaNodePool ... if I am starting from node pools?"
Then I realized you mean that if you have multiple node pools you should move to have just one. Can we make it clearer? Saying something like "Be sure to have just one node pool ...", "If you have multiple node pools please move to a single one ...". Don't know something to make it clearer.

Reply:
Okay. I've rewritten as follows:

> If you have multiple node pools, consolidate them into a single `KafkaNodePool` named `kafka` with node IDs from 0 to N (where N is the number of replicas).